docs/guides/index.md

```diff
@@ -7,10 +7,14 @@ tags:
     - how to
 ---
 
+!!! tip "Need help?"
+    Get your questions answered with a Prefect product advocate by [Booking A Rubber Duck](https://calendly.com/prefect-experts/prefect-product-advocates?utm_campaign=prefect_docs_cloud&utm_content=prefect_docs&utm_medium=docs&utm_source=dpcs)!
+
```

Comment:
My thought is that this would flow more nicely if it were after the bulleted list of guides (ln 21-ish). 

@billpalombi thoughts on the placement of the one in cloud?

Reply:
I agree that it would fit better after the list.

Reply:
I feel like the bulleted list would indicate that it is the same as the other items, which it is a significant departure from. Down to move the cloud one down.